markasread/options.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html>
+	<head>
+		<title>Mark As Read</title>
+		<link rel="stylesheet" type="text/css" href="options.css">
+		<script src="options.js"></script>
+	</head>
+	<body>
+		<h1>Mark As Read</h1>
+		<section>
+			<h3>Change Color of Visited Links</h3>
+			<div class="row">
+				<label for="changeLinkColor">Enabled</label>
+				<input id="changeLinkColor" type="checkbox" />
+			</div>
+			<div class="row">
+				<label for="linkColor">Link Color</label>
+				<input id="linkColor" type="text" placeholder="(default = blue) : [red, yellow, magenta]" size="43"/>
```

Comment:
<img width="508" alt="Screenshot 2020-04-20 at 09 43 10" src="https://user-images.githubusercontent.com/973854/79729877-551dce00-82f0-11ea-95aa-933153b0bb96.png">
<img width="304" alt="Screenshot 2020-04-20 at 09 43 17" src="https://user-images.githubusercontent.com/973854/79729900-5949eb80-82f0-11ea-8b70-bfbf47c94428.png">

This should look like this.